source/renderer/app/components/staking/delegation-center/WalletRow.scss

```diff
@@ -114,8 +114,7 @@
 
       .stakePoolTicker {
         @extend %accentText;
-        font-size: 14px;
-        letter-spacing: -0.5px;
+        @extend %stakePoolTicker;
         line-height: 1.57;
```

Comment:
Just curious if our code style describes this "@extend" position. Here both "extends" at the beginning, later in your changes you use them in the end

Reply:
Good point, seems like this is not linted atm 👍

Reply:
@alexander-rukin i created a separate issue for improving the stylelint config to include "at-rules": https://input-output.atlassian.net/browse/DDW-993